run-rafi.sh

```
@@ -0,0 +1,421 @@
+#    if ! mvn clean install; then
+#        echo -e "${RED}Build failed. Fix the errors and try again.${NC}"
+#        exit 1
+#    fi
+#    echo -e "${GREEN}Build completed successfully.${NC}"
+# }
+build_app() {
+   echo -e "${CYAN}Building the application...${NC}"
+   # Load environment variables before building
+   if [ -f .env ]; then
+       echo -e "${CYAN}Loading environment variables from .env file...${NC}"
+       export $(cat .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)